server/src/configureAPI.mjs

```diff
@@ -77,6 +77,11 @@ function configureAPI(app) {
 
     // End a session
     app.post('/api/v1/session/stop',                   sessionApi.stopSession);
+
+    // ======================= Sequence-Related API Routes =======================
+
+    // Send a sequence
```

Comment:
Minor: Please change this to "Send an event sequence" to be more explicit. Same with line 81... add "Event " before "Sequence".